docs/authentication/social-connections/tiktok.mdx

```diff
@@ -1,47 +1,84 @@
+It is recommended to test this integration in a staging or preview environment, as shared credentials require URL verification to function properly.
 
-## Before you start
+## Configure for your production instance
 
-- You need to create a Clerk Application in your [Clerk Dashboard](https://dashboard.clerk.com/). For more information, check out our [Set up your application guide](/docs/quickstarts/setup-clerk).
-- You need to have a TikTok Developer account. To create one, visit the [TikTok for developers](https://developers.tiktok.com/) page and click on Login ➜ Sign Up
+To make the setup process easier, it's recommended to keep two browser tabs open: one for your [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections) and one for your [TikTok Developer Portal](https://developers.tiktok.com/).
 
-## Configuring TikTok social connection
+<Steps>
+  ### Enable TikTok as a social connection in Clerk
 
-First, you need to create a new TikTok app. Go to the [TikTok for developers](https://developers.tiktok.com/) page, login with your developer account and create a new app by clicking on the **My Apps** ➜ **Connect a new app** and following the on screen wizard.
+  1. In the Clerk Dashboard, navigate to the [**SSO Connections**](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections) page.
+  1. Select **Add connection** and select **For all users**.
+  1. In the **Choose provider** dropdown, select **TikTok**.
+  1. Ensure that both **Enable for sign-up and sign-in** and **Use custom credentials** are toggled on. Keep the modal and page open.
 
-![Creating a new TikTok app](/docs/images/authentication-providers/tiktok/9a5749a9bf3f282c54af2a2b539ba372e4706574-1489x964.png)
+  ### Create your app in TikTok
 
-Add an icon and a name for your new project and hit **Start**.
+  > [!TIP]
+  > If you're creating an app as an organization rather than as an individual developer, you must first [create an organization](https://developers.tiktok.com/organizations). For guidance on which option to choose, refer to [TikTok's guide on working with organizations](https://developers.tiktok.com/doc/working-with-organizations/)**.**
 
-You'll get redirected to the app creation form. Notice that you need to fill the **Callback URL** and **Redirect domain** fields.
+  1. On a separate page, go to the [TikTok Developer Portal](https://developers.tiktok.com/) and sign in.
+  1. In the top-right, select [**Developer Portal**](https://developers.tiktok.com/apps), then select **Manage apps**. You'll be redirected to the **Manage apps** page.
+  1. Select **Connect an app**. Complete the form then select **Confirm**. You'll be redirected to your **App details** page.
+  1. In the **Basic information** section, complete the form.
+  1. Under **Platforms**, select your platform then paste your app URL.
```

Comment:
These is handled in the step before "Complete the form" this is part of completing the form

```suggestion
```